platform/ios/src/MGLMapView.mm

```diff
@@ -564,6 +565,8 @@ - (void)commonInit
     if ([UIApplication sharedApplication].applicationState != UIApplicationStateBackground) {
         [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
     }
+
+    [MGLSDKUpdateChecker checkForUpdates];
```

Comment:
I also like the idea of calling this from the `load` or `initialize` class methods. If you do move it, calling from `initialize` [like Realm do](https://github.com/realm/realm-cocoa/blob/8d48a1dd6430f8ed5e30f7fe5125e999c1b24465/Realm/RLMRealm.mm#L136) may be slightly safer since this ends up making a network call but probably fine either way.